src/bitcoind/interface.rs

```diff
@@ -34,6 +28,9 @@ pub struct BitcoinD {
     node_client: Client,
     watchonly_client: Client,
     cpfp_client: Client,
+
+    /// The number of try a client is allowed to do for a request.
+    try_limit: usize,
```

Comment:
Nit: i think it's more intuitive for it to be the number of retries?
```suggestion
    /// How many times we'll try again to send a request to bitcoind upon failure
    retries: usize,
```